client/my-sites/media/main.jsx

```diff
@@ -358,6 +359,20 @@ class Media extends Component {
 				{ mediaId && site && site.ID && <QueryMedia siteId={ site.ID } mediaId={ mediaId } /> }
 				<PageViewTracker path={ this.getAnalyticsPath() } title="Media" />
 				<DocumentHead title={ translate( 'Media' ) } />
+				{ this.props.selectedSite.is_private && this.props.selectedSite.is_wpcom_atomic && (
+					<Notice
+						showDismiss={ false }
+						status="is-info"
+						text={ translate(
+							'Your site is Private and the image CDN is disabled. If image thumbnails do not display, switch to Coming Soon mode. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
```

Comment:
```suggestion
							'The image CDN is disabled because your site is marked Private. If image thumbnails do not display in your Media Library, you can switch to Coming Soon mode. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
```

Reply:
Thank you for that suggestion! Done!